tutorials/scripting/evaluating_expressions.rst

```diff
@@ -45,6 +45,7 @@ The following operators are available:
 |                        | If at least one of them is a floating-point number, returns a floating-point value. |
 +------------------------+-------------------------------------------------------------------------------------+
 | Modulo (``%``)         | Returns the remainder of an integer division.                                       |
+|                        | The result will always have the sign of the dividend.                               |
```

Comment:
Could be improved further:

```suggestion
| Remainder (``%``)      | Returns the remainder of an integer division.                                       |
|                        | The result will always have the sign of the dividend.                               |
```

The behavior in GDScript is Remainder, not Modulo, which is what Python's `%` operator does.

Reply:
That this page uses "modulo" and the other (linked by @dalexeev ) uses "remainder" led to me only finding this and not the other when I searched – terminology around this is often a bit fuzzy. Even if one is more technically accurate than the other, it might be useful to have both in docs to make sure it is relatively searchable.